api/HTMLElement.json

```diff
@@ -868,6 +868,67 @@
           }
         }
       },
+      "enterKeyHint": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
+            },
+            "chrome_android": {
+              "version_added": "77"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "79",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.forms.enterkeyhint",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "79",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.forms.enterkeyhint",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "13.1"
```

Comment:
I notice that https://webkit.org/blog/10247/new-webkit-features-in-safari-13-1/ says (emphasis added):

> **On iOS**, WebKit supports the `enterkeyhint` attribute

That seems to imply it’s *not* supported in desktop Safari?

Reply:
DOM binding supports on both macOS and iOS (https://bugs.webkit.org/show_bug.cgi?id=203440).  I tested on wpt.live on Safari/macOS.
UX support for software keyboard is https://bugs.webkit.org/show_bug.cgi?id=189546. This is iOS only.

Reply:
> DOM binding supports on both macOS and iOS (https://bugs.webkit.org/show_bug.cgi?id=203440). I tested on wpt.live on Safari/macOS.

OK, so then I think it’s clear that marking it as macOS-supported (as this PR does) is accurate.

> UX support for software keyboard is https://bugs.webkit.org/show_bug.cgi?id=189546. This is iOS only.

So should we add a note field for the `safari` data that says something like:

> "note": "Doesn’t expose UX support (software keyboard is exposed only on iOS)."

…or something like that?

Reply:
> So should we add a note field for the safari data that says something like:
>> "note": "Doesn’t expose UX support (software keyboard is exposed only on iOS)."
> …or something like that?

This issue is same as Chrome desktop since Android and iOS has the OS API to change `[enterkey]`, but no API for others such as Windows.  Should we add same note to chrome desktop too?

Reply:
I would expect this to be true of all desktop platforms, not just Safari. This is a similar situation to some media query never matching on some platforms. Do we have notes for other such situations? If we can't foresee that this attribute would ever do anything on desktop platforms, then such notes don't seem useful.

Reply:
> This issue is same as Chrome desktop since Android and iOS has the OS API to change `[enterkey]`, but no API for others such as Windows. Should we add same note to chrome desktop too?

I’m not sure what the BCD policy is for these kinds of cases. I am sure we have some precedents, but I’m not sure what we did on those cases. I guess that in general we have a number of features that we all known aren’t really meant to be used on desktop — because no UX is exposed for them on desktop, even though the underlying APIs are exposed. So in those cases, we shouldn’t mark the engine support as missing.

Anyway, I’ve opened https://github.com/mdn/browser-compat-data/issues/6914 to track that issue for this case. But I don’t think we need to block this PR on waiting for resolution of that — instead we can just do a follow-up PR later.

So I’ll go ahead and merge this PR.